api/prisma/schema.prisma

```diff
@@ -281,20 +281,31 @@ enum UnterveranstaltungType {
 }
 
 model Unterveranstaltung {
-  id               Int                    @id @default(autoincrement())
+  id               Int                          @id @default(autoincrement())
   maxTeilnehmende  Int
   teilnahmegebuehr Int
   meldebeginn      DateTime
   meldeschluss     DateTime
   veranstaltungId  Int
-  veranstaltung    Veranstaltung          @relation(fields: [veranstaltungId], references: [id], onDelete: Cascade)
+  veranstaltung    Veranstaltung                @relation(fields: [veranstaltungId], references: [id], onDelete: Cascade)
   gliederungId     Int
-  gliederung       Gliederung             @relation(fields: [gliederungId], references: [id], onDelete: Cascade)
+  gliederung       Gliederung                   @relation(fields: [gliederungId], references: [id], onDelete: Cascade)
   Anmeldung        Anmeldung[]
   beschreibung     String?
   bedingungen      String?
-  type             UnterveranstaltungType @default(GLIEDERUNG)
+  type             UnterveranstaltungType       @default(GLIEDERUNG)
   customFields     CustomField[]
+  documents        UnterveranstaltungDocument[]
+}
+
+model UnterveranstaltungDocument {
```

Comment:
Wozu diese extra Entität? Verknüpfung Unterveranstaltung -> File reicht in meinen Augen, dann als Array einfach

Reply:
Der Gedanke hier ist, dass Dokumente auch einen beschreibenden Namen enthalten kann. Theoretisch könnte man das auch in File packen.